client/src/pages/NotFound.jsx

```diff
@@ -0,0 +1,13 @@
+import { Box } from "@mui/material";
+import Typography from "../Mui-Components/Typography";
+
+const ContactUs = () => {
+	return (
+		<Box sx={{ textAlign: "center", p: 15 }}>
+			<Typography variant="h1">Error 404</Typography>
+			<Typography variant="h2">Page Not Found!</Typography>
+		</Box>
+	);
+};
```

Comment:
does this need to be a function? Can't it just be

```javascript
const ContactUs = 
		<Box sx={{ textAlign: "center", p: 15 }}>
			<Typography variant="h1">Error 404</Typography>
			<Typography variant="h2">Page Not Found!</Typography>
		</Box>
```

Reply:
It didn't work because it's in another component and needs to be returned.